app/decorators/prosecution_case_decorator.rb

```diff
@@ -23,4 +21,25 @@ def hearings_with_day_by_datetime
   def cracked?
     hearings.map { |h| h.cracked_ineffective_trial&.cracked? }.any?
   end
+
+  def column_sort_icon
+    direction == 'asc' ? "\u25B2" : "\u25BC"
+  end
+
+  def column_title(column)
+    case column
+    when 'type'
+      t('search.result.hearing.hearing_type')
+    when 'provider'
+      t('search.result.hearing.providers')
+    else
+      t('search.result.hearing.hearing_day')
+    end
+  end
```

Comment:
**not a blocker**

A more OO alternative
```
def column_title
  Hash.new(t('search.result.hearing.hearing_day')).merge(
    type: t('search.result.hearing.hearing_type'),
    provider: t('search.result.hearing.providers')
  )[column]
end
```
Column is attribute on the object anyway I believe, so does it need passing to it. Although you may, and probably should, handle nil scenario for the attribute in here, rather than in the controller.

Reply:
@jsugarman, followed the suggestion to use a hash rather than a case statement here. I am also now handling the defaults for hearings_sort_column and hearings_sort_direction here rather than in the controller.